openvpn-install.sh

```diff
@@ -22,7 +25,7 @@ fi
 if [[ -e /etc/debian_version ]]; then
 	OS="debian"
 	# Getting the version number, to verify that a recent version of OpenVPN is available
-	VERSION_ID=$(cat /etc/os-release | grep "VERSION_ID")
+	VERSION_ID=$(grep "VERSION_ID" < /etc/os-release)
```

Comment:
`<` is not needed with `grep`.

Reply:
I see. This can be removed later if PR get merged.